packages/shared/providers.yaml

```diff
@@ -3237,20 +3255,23 @@ keeper-scim:
             method: GET
             endpoint: /Users
     docs: https://docs.nango.dev/integrations/all/keeper
+    docs_connect: https://docs.nango.dev/integrations/all/keeper/connect
     credentials:
         apiKey:
             type: string
             title: API Key
             description: The API key for your node keeper account
             pattern: '^[A-Za-z0-9+/=]+$'
             example: 'J9b7kgdL2gf14d5F9p67zYXVrTZPTMEnl3/EmwwI9K2='
+            doc_section: '#step-1-finding-your-api-key-api-token-and-node-id'
     connection_config:
         node:
             type: string
             title: Node
             description: The node id to your Keeper account
             pattern: '^\d{15}$'
             example: '123435384338765'
+            doc_section: '#step-1-finding-your-api-key-api-token-and-node-id'
```

Comment:
A comment that applies for all markdown, it should be one section per entry so the doc_section is directly relevant and ordered by the field order, e.g:
- Step 1: find your node id
- Step 2: find your api key

NB: you can reorder fields based on what's more logical, here for example apiKey will be first because we put credentials first but it's more logical to put `order: 1`

Reply:
> A comment that applies for all markdown, it should be one section per entry so the doc_section is directly relevant and ordered by the field order, e.g:

> Step 1: find your node id
Step 2: find your api key

this has been updated and should be in the correct order.